apps/src/craft/code-connection/cc-client.js

```diff
@@ -18,19 +35,23 @@ export default class cc_client {
      *
      */
     connectionStatusUpdate(asyncCallback, timeout = 200) {
-        $.ajax({
-            type: "GET",
-            url: `${baseUrl}${this.port}/connected`,
-            timeout: timeout,
-            success: function (data) {
-                // cc responded
-                asyncCallback(data);
-            },
-            error: function (jqxhr, textStatus, error) {
-                // TODO: handle net::ERR_CONNECTION_REFUSED error gracefully
-                asyncCallback(false);
-            }
-        });
+        if (window.ipcRenderer !== undefined) {
+            asyncCallback(window.ipcRenderer !== null);
```

Comment:
are `undefined` and `null` both valid and expected values for `ipcRenderer`?

Reply:
Apparently not all browsers treat them the same. They should really be undefined. Looks like my last commit missed one!